docs/standard/assembly/identify.md

```diff
@@ -37,7 +37,7 @@ The <xref:System.Reflection.AssemblyName.GetAssemblyName%2A> method loads the te
 
 ### Using the PEReader class
 
-1. Install the [System.Reflection.Metadata](https://www.nuget.org/packages/System.Reflection.Metadata/) NuGet package.
+1. When targeting .NET Standard or .NET Framework, install the [System.Reflection.Metadata](https://www.nuget.org/packages/System.Reflection.Metadata/) NuGet package. When targeting .NET Core or .NET 5+, this step is not required, because this library is included in the shared framework.
```

Comment:
```suggestion
1. When targeting .NET Standard or .NET Framework, install the [System.Reflection.Metadata](https://www.nuget.org/packages/System.Reflection.Metadata/) NuGet package. When targeting .NET Core or .NET 5+, this step isn't required because this library is included in the shared framework.
```